azure-pipelines.yml

```diff
@@ -32,7 +32,13 @@ jobs:
       ## manually call this since lage is now used and it calls lint per package, not per repo
       - script: |
           yarn prelint
-          yarn prettier $(targetBranch) --check
+
+          if [ $BUILD_SOURCEBRANCH == "refs/heads/master" ]; then
```

Comment:
@ecraig12345 I am not sure about casing there (the original variable is `Build.SourceBranch`), but according to Azure docs, that's correct: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#systemaccesstoken

Reply:
Yes `BUILD_SOURCEBRANCH` should be correct for shell script access. There are too many ways to access variables... This page kind of helps explain it. https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops